### PR TITLE
Basic security policy

### LINKED ISSUE #3: Security Policy violation SECURITY.md
_This issue was automatically created by [Allstar](https://github.com/ossf/allstar/)._

**Security Policy Violation**
Security policy not enabled.
A SECURITY.md file can give users information about what constitutes a vulnerability and how to report one securely so that information about a bug is not publicly visible. Examples of secure reporting methods include using an issue tracker with private issue support, or encrypted email with a published key.

To fix this, add a SECURITY.md file that explains how to handle vulnerabilities found in your repository. Go to https://github.com/kubereboot/.github/security/policy to enable.

For more information, see https://docs.github.com/en/code-security/getting-started/adding-a-security-policy-to-your-repository.

---

This issue will auto resolve when the policy is in compliance.

Issue created by Allstar. See https://github.com/ossf/allstar/ for more information. For questions specific to the repository, please contact the owner or maintainer.

### LINKED ISSUE #14: Security Policy violation SECURITY.md
_This issue was automatically created by [Allstar](https://github.com/ossf/allstar/)._

**Security Policy Violation**
Security policy not enabled.
A SECURITY.md file can give users information about what constitutes a vulnerability and how to report one securely so that information about a bug is not publicly visible. Examples of secure reporting methods include using an issue tracker with private issue support, or encrypted email with a published key.

To fix this, add a SECURITY.md file that explains how to handle vulnerabilities found in your repository. Go to https://github.com/kubereboot/website/security/policy to enable.

For more information, see https://docs.github.com/en/code-security/getting-started/adding-a-security-policy-to-your-repository.

---

This issue will auto resolve when the policy is in compliance.

Issue created by Allstar. See https://github.com/ossf/allstar/ for more information. For questions specific to the repository, please contact the owner or maintainer.

### LINKED ISSUE #15: Security Policy violation SECURITY.md
_This issue was automatically created by [Allstar](https://github.com/ossf/allstar/)._

**Security Policy Violation**
Security policy not enabled.
A SECURITY.md file can give users information about what constitutes a vulnerability and how to report one securely so that information about a bug is not publicly visible. Examples of secure reporting methods include using an issue tracker with private issue support, or encrypted email with a published key.

To fix this, add a SECURITY.md file that explains how to handle vulnerabilities found in your repository. Go to https://github.com/kubereboot/charts/security/policy to enable.

For more information, see https://docs.github.com/en/code-security/getting-started/adding-a-security-policy-to-your-repository.

---

This issue will auto resolve when the policy is in compliance.

Issue created by Allstar. See https://github.com/ossf/allstar/ for more information. For questions specific to the repository, please contact the owner or maintainer.

### LINKED ISSUE #667: Security Policy violation SECURITY.md
_This issue was automatically created by [Allstar](https://github.com/ossf/allstar/)._

**Security Policy Violation**
Security policy not enabled.
A SECURITY.md file can give users information about what constitutes a vulnerability and how to report one securely so that information about a bug is not publicly visible. Examples of secure reporting methods include using an issue tracker with private issue support, or encrypted email with a published key.

To fix this, add a SECURITY.md file that explains how to handle vulnerabilities found in your repository. Go to https://github.com/kubereboot/kured/security/policy to enable.

For more information, see https://docs.github.com/en/code-security/getting-started/adding-a-security-policy-to-your-repository.

---

This issue will auto resolve when the policy is in compliance.

Issue created by Allstar. See https://github.com/ossf/allstar/ for more information. For questions specific to the repository, please contact the owner or maintainer.

### PR DESCRIPTION
**Pull request by dholbach**
Fixes: #3 kubereboot/charts#15 kubereboot/website#14 kubereboot/kured#667 

Signed-off-by: Daniel Holbach <daniel@weave.works>

### CODE REVIEW

**Review by ckotzbauer — APPROVED** [review]

**Comment by dholbach** [comment]
Let's agree on this together. @jackfrancis @evrardjp How do you feel about the doc as a first start?

Both sections are essentially what GitHub provided in a small template, so I went ahead and filled it with what forms our reality today.

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Security Policy
+
+## Supported Versions
+
+Kured follows the following version scheme:
```
Comment (jackfrancis) [inline SECURITY.md]: "follows the following" has a bit of a stutter. We can probably link to the semver spec. Maybe something like:

Kured releases versions in accordance with the [Semantic Version spefication](https://semver.org):

**Comment by jackfrancis** [comment]
left a comment, overall lgtm, thanks @dholbach!

**Review by jackfrancis — APPROVED** [review]
/lgtm

**Comment by dholbach** [comment]
@evrardjp wdyt?

**Comment by dholbach** [comment]
Ok, let's merge for now and improve in follow-up PRs.

**Pull request merged**